site/src/user.js

```diff
@@ -1,17 +1,33 @@
 import { writable } from 'svelte/store';
 
+
 export const user = writable(null);
 
 if (process.browser) {
-	// TODO this is a workaround for the fact that there's currently
-	// no way to pass session data from server to client
-	// TODO there is now! replace this with the session mechanism
-	fetch('/auth/me.json', { credentials: 'include' })
-		.then(r => r.json())
-		.then(user.set);
+	const storageKey = 'svelte-dev:token';
+
+	// On load, get the last-known user token (if any)
+	// Note: We can skip this all by writing User data?
```

Comment:
We already have the complete user object. We _could_ write it to localStorage and have the user-data immediately available. If/when this is the case, we fetch the `/auth/me.json` endpoint which will wipe/reset if the token was expired or invalid.